### DISCUSSION TITLE
Moving all code related stuff into subfolder

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#855] [Ideas]
I am considering moving all the project code relate stuff into subfolder. Here's why...

When you open current project page, you get this view:

![ss](https://user-images.githubusercontent.com/8041294/128428031-d329d806-4e6a-4e4a-8e9a-bfe889be011f.png)

If you are a developer, that's nothing that's bother you, but if you are not a dev, but just look for a tool then that page can be extremely confusing. Tons of odd links and folders and who knows what else. And nothing that you really need to bother at first. If you just need to get a tool for your job/class then all this give you one big WTF. 

Let's then imagine we hide all technical stuff (incl. `board_models` folder) and move it to subfolder. Then, landing page looks more like this:

![ss3](https://user-images.githubusercontent.com/8041294/128429328-55ffc654-9f9f-468a-ba8d-7506eece4b92.png)

So what's the price? Current devs would need to adjust the paths **once** (new devs will just set it up for new location). Github Actions needs to be updated to look for configs in new paths. Same for other tools - trivial docs update needed. And that's it. It's pretty cheap if you ask me and we get more "customer friendly" project landing page. 

Thoughts?

@BFH-ktt1 @maehne @davidhutchens @R3dst0ne

**Comment by R3dst0ne** [comment]
GitHub is mainly aimed at developers, and Logisim-evolution is a software that is not likely used by newbies. Thus, I don't see a problem with having a programmer friendly layout.
I'm for creating a `gradle` folder, since ~1/3 of all files in the root directory are related to it.

**Reply by MarcinOrlowski** [reply]
Github page is our only page. There's no point of moving just gradle stuff. For users gradle is as cryptic as all other stuff so either we move all to subfolder (which makes it easier to handle as you just move code root folder there) or nothing at all.

**Reply by maehne** [reply]
I agree with @R3dst0ne: The GitHub page should target primarily developers and they are used that they see the top of the project folder. For users, we should create a proper homepage at `https://logisim-evolution.github.io/`. This has only become possible with our move to a dedicated Github organisation. We can discuss efforts in this direction, when we'll discuss next week the Roadmap.
